cookbook/attributes/default.rb

```diff
@@ -10,7 +10,7 @@
   :user => 'singularity',
   :group => 'singularity',
   :git_ref => 'e2405eb5ca1a1ba006a89a27bdb3299433ae96d5',
-  :version => '0.4.2',
+  :version => '0.4.1',
```

Comment:
does `git_ref` need to be updated in response to this change? could we do one better and use a tag name instead of a legit SHA?